gprofiler/profilers/python.py

```diff
@@ -189,7 +192,7 @@ def _make_command(self, pid: int, output_path: str, duration: int) -> List[str]:
             "--format",
             "raw",
             "-F",
-            "--gil",
+            # "--gil",
```

Comment:
Not working on Windows? In any case, disable only for the Windows branch, not for Linux

Reply:
Done